client/packages/system/src/Item/api/hooks/useItemVariant/useItemVariantsConfigured.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-query';
+import { useItemGraphQL } from '../useItemApi';
+import { ITEM_VARIANTS } from '../../keys';
+
+export const useItemVariantsConfigured = () => {
```

Comment:
Naming-wise, `useIsItemVariantsConfigured` is more consistent with other similar hooks -- or `useAreItemVariantsConfigured` if we want to be grammatically correct 🤷‍♂️